js/user.js

```diff
@@ -1,15 +1,35 @@
 const urlBase = 'LAMPAPI/';
 const extension = '.php';
 
+let firstName = "";
+let lastName = "";
+let fullName = "";
+let id = -1;
+
 // Tries to autologin a user. If it can't then it loads the hi res background
 window.onload = function () {
+	// For local containers
+	// const dom = "http://localhost/";
 	const dom = "https://contactical.xyz/";
-	if ((window.location.href === dom.concat("index.html") ||
-		window.location.href === dom) && autoLogin())
-		return;
+	if (window.location.href === dom.concat("index.html") || window.location.href === dom) {
+		if (autoLogin())
+			return;
+	}
+	else if (window.location.href !== dom.concat("create_account.html")) {
+		populateUserCache();
+		welcomeUser();
+	}
```

Comment:
It's usually good practice to only use `window.onload` if the js file is only associated with another single html file, not multiple. In our case we can have an `index.js` for `index.html` and a `landing_page.js` for `landing_page.html`. Then each of them will have their own `window.onload()`. This will prevent us from having to check which page we are on because with the coupled js files is always known. I.E when I'm in `index.js` I know for a fact that the page I'm on is `index.html`, this holds true with the landing page as well.

Reply:
That's definitely a much cleaner way to handle things. I will take care of it now :)